apps/reference/nav/_referenceSidebars.js

```diff
@@ -173,7 +173,13 @@ const sidebars = {
         'guides/realtime',
         'guides/realtime/quickstart',
         'guides/realtime/postgres-cdc',
-      ],
+          {
+            type: 'doc',
+            id: 'guides/realtime/limits', // document ID
```

Comment:
If you decide to change the id to `rate-limits`:

```suggestion
            id: 'guides/realtime/rate-limits', // document ID
```

Reply:
And don't forget to rename the file to `apps/reference/docs/guides/realtime/rate-limits.mdx`.